src/main/kotlin/frc/robot/subsystems/elevator/ElevatorIOReal.kt

```diff
@@ -0,0 +1,65 @@
+package frc.robot.subsystems.elevator
+
+import MOTOR_ID
+import com.ctre.phoenix6.configs.CurrentLimitsConfigs
+import com.ctre.phoenix6.configs.FeedbackConfigs
+import com.ctre.phoenix6.configs.MotorOutputConfigs
+import com.ctre.phoenix6.configs.Slot0Configs
+import com.ctre.phoenix6.configs.TalonFXConfiguration
+import com.ctre.phoenix6.controls.DutyCycleOut
+import com.ctre.phoenix6.controls.PositionVoltage
+import com.ctre.phoenix6.hardware.TalonFX
+import com.ctre.phoenix6.signals.InvertedValue
+import com.ctre.phoenix6.signals.NeutralModeValue
+import edu.wpi.first.units.Units
+import edu.wpi.first.units.measure.Distance
+import kotlin.math.PI
+
+class ElevatorIOReal : ElevatorIO {
+    override val inputs = LoggedElevatorInputs()
+    private val motor = TalonFX(MOTOR_ID)
+    private val motorPosititonRequest = PositionVoltage(0.0)
+
+    init {
+        val motorConfig = TalonFXConfiguration().apply {
+            MotorOutput = MotorOutputConfigs().apply {
+                NeutralMode = NeutralModeValue.Brake
+                Inverted = InvertedValue.Clockwise_Positive
+            }
+            Feedback = FeedbackConfigs().apply {
+                RotorToSensorRatio = 1.0
+//                SensorToMechanismRatio = GEAR_RATIO * FIRST_STAGE_RATIO
```

Comment:
Remove the comment if this is not required.

Reply:
48f8665322c69bae5f76ec0a5cf165521b2592e8

Reply:
48f8665322c69bae5f76ec0a5cf165521b2592e8